<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeSetDialog</class>
 <widget class="QDialog" name="NodeSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IP：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mask">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mask：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_number">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Number：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_protocol">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Protocol：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>dynamic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>static</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mask_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mask_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mask_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mask_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NodeSetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NodeSetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
